src/components/contentWrapper/ContentWrapper.tsx

```diff
@@ -19,6 +19,7 @@ export const ContentWrapper = ({
 }: IOContentWrapperProps) => (
   <View
     style={{
+      flex: 1,
```

Comment:
```suggestion
```
Tried to delete it and nothing changed. In general, we shouldn't change a layout prop in a generic component like `ContentWrapper` because it's quite risky.

Reply:
this was due to some testing, we can revert with no problem